src/functions.jl

```diff
@@ -836,6 +836,26 @@ end
 ### Base.convert
 ###
 
+"""
+    conversion_cost(F::Type{<:MOI.AbstractFunction}, G::Type{<:MOI.AbstractFunction})
+
+Return a `Float64` returning the *cost* of converting any function of type `G`
+to a function of type `F` with `convert`. This cost is used to compute
+[`Bridges.bridging_cost`](@ref).
+This cost is `Inf` should be used to avoid the
+[`Bridges.Constraint.FunctionConversionBridge`](@ref) to attempt the conversion.
+"""
+function conversion_cost(::Type{<:AbstractFunction}, ::Type{<:AbstractFunction})
```

Comment:
This feels like it should be a method in the bridges, not in MOI in general. It's  weird special case that isn't generally applicable, and the values are arbitrary.

Reply:
Yes, we could move it to the bridges